Meta/Lagom/CMakeLists.txt

```diff
@@ -472,7 +500,6 @@ if (BUILD_LAGOM)
     if(${SDL2_FOUND})
         file(GLOB LIBGUI_LAGOM_SOURCES CONFIGURE_DEPENDS "./Libraries/LibGUI/*.cpp")
         file(GLOB LIBGUI_SOURCES CONFIGURE_DEPENDS "../../Userland/Libraries/LibGUI/*.cpp")
-        file(GLOB LIBGUI_SUBDIR_SOURCES CONFIGURE_DEPENDS "../../Userland/Libraries/LibGUI/Lagom/*.cpp")
```

Comment:
If this shouldn't have been introduced, it should be squashed into the commit that introduced it rather than removed here.

Reply:
Will fix